### PR TITLE
NAS-109153 / 21.02 / Implemented entity-empty on entity-table

### PR DESCRIPTION
**Pull request by ryousaf**
NAS-109153

### CODE REVIEW

File: src/app/pages/common/entity/entity-table/entity-table.component.ts [hunk 1]
```diff
@@ -560,6 +592,35 @@ export class EntityTableComponent implements OnInit, AfterViewInit, OnDestroy {
...
+    if(this.currentRows && this.currentRows.length > 0) {
+      this.isTableEmpty = false;
+    } else {
+      this.isTableEmpty = true;
+      if(this.firstUse) {
+        this.emptyTableConf = {
+          type: EmptyType.first_use,
+          large: true,
+          title: "No "+this.title,
+          message: `It seems you haven't setup any ${this.title} yet.`
```
Comment (erinix) [inline src/app/pages/common/entity/entity-table/entity-table.component.ts:604]: these strings are not translateable, consider splitting up and adding T() to the text

**Review by erinix — CHANGES_REQUESTED** [review]

File: src/app/pages/common/entity/entity-table/entity-table.component.ts (hunk 1 above)
Comment (ryousaf) [reply src/app/pages/common/entity/entity-table/entity-table.component.ts]: Fixed strings

**Comment by dszidi** [comment]
Found some things with entity-empty template and css that needed to be fixed.
Here is a PR against this branch...

https://github.com/freenas/webui/pull/5127

**Review by dszidi — APPROVED** [review]
lgtm

**Review by erinix — APPROVED** [review]

**Pull request merged**